### ISSUE TITLE
does it works on oreo?

### ISSUE THREAD

**Issue by ArslanKathia** [alhazmy13/MediaPicker#82]
attach the video or image from gallery works properly in oreo 8.0 or above version..

**Comment by mikemike396** [comment]
Yes, it's working for us. This PR fixed issues introduced in 8.0+

https://github.com/alhazmy13/MediaPicker/pull/73
